rpm/kubelet.spec

```diff
@@ -17,7 +17,7 @@
 %global CNI_VERSION 0.5.1
 %endif
 
-%global CRI_TOOLS_VERSION 1.11.1
+%global CRI_TOOLS_VERSION 1.12.0
```

Comment:
shouldn't we control CRI_TOOLS_VERSION based on the KUBE_SEMVER, like CNI_VERSION is above?

(this is also another strong argument in favor of keeping all rpm and deb specs in kubernetes/kubernetes rather than kubernetes/release.)

Reply:
ohhhhh boy. Yeah....